bedrock/base/templates/base-pebbles.html

```diff
@@ -102,9 +102,16 @@
             <li class="first about-item">
               <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
             </li>
+            {% if LANG.startswith('en-') %}
```

Comment:
Would this be better as `l10n_has_tag` (here and below), or would the `.lang` file dance be too convoluted?